tahrir/endpoints/admin/__init__.py

```diff
@@ -0,0 +1,4 @@
+from flask import Blueprint
+
+
+blueprint = Blueprint("admin_endpoints", __name__)
```

Comment:
You can just name it "admin", blueprints always contain endpoints. (or maybe "api_admin"?)